docs/cudf/source/guide-to-udfs.ipynb

```diff
@@ -22,7 +22,7 @@
     "- CuPy NDArrays\n",
```

Comment:
<p>As above, I think this section can just be <code>N/A support</code>  without references to the behavior in previous versions. With time, previous behaviour will become less relevant with time and likely we won't get a chance to update the docs.</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/rapidsai/cudf/pull/8810/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='49'/>